.test-infra/terraform/google-cloud-platform/google-kubernetes-engine/cluster.tf

```diff
@@ -34,7 +34,9 @@ resource "google_container_cluster" "default" {
     enable_private_nodes    = true
     enable_private_endpoint = false
   }
-  node_config {
-    service_account = data.google_service_account.default.email
+  cluster_autoscaling {
```

Comment:
Thank you for changing this! Could we also add the oauth scope for Google cloud platform? https://www.googleapis.com/auth/cloud-platform

Reply:
What happened when you tested this terraform module then? Were you able to successfully deploy the kafka cluster? The autopilot terraform specification changes rapidly but I had some success recently with:
```
resource "google_container_cluster" "default" {
  name                = var.resource_name
  deletion_protection = false
  location            = var.region
  enable_autopilot    = true
  network             = google_compute_network.default.id
  subnetwork          = google_compute_subnetwork.default.id
  private_cluster_config {
    enable_private_nodes    = true
    enable_private_endpoint = false
  }
  cluster_autoscaling {
    auto_provisioning_defaults {
      service_account = google_service_account.default.email
      oauth_scopes    = [
        "https://www.googleapis.com/auth/cloud-platform"
      ]
    }
  }
}
```

Reply:
Forgive me not being clear but code from my comment is fine now